### PR TITLE
Add individual and groups contribution budget

### PR DESCRIPTION
**Pull request by jestradadeveloper**
# Description
Add method to user an group models in order to get the spent or contribution of a participant in a group

What did you implemented/Why did you implemented this:
 - get individual contribution 
 - get budget by owned groups and participation groups
 - annotate initial and default setting
 - annotation of models

  ## Screenshots
Captura de Pantalla 2021-11-30 a la(s) 18 45 57
Captura de Pantalla 2021-11-30 a la(s) 18 46 04

### CODE REVIEW

**Review by JuanequeX — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -67,3 +67,4 @@ Incluyan en [esta carpeta](/design) su propuesta de diseño
 - Colocar variables de entorno correspondientes copiando .env.example en un nuevo archivo .env
 - Ejecutar rails db:create
 - Si no es la primera vez asegurarse de realizar las migraciones con: rails db:migrate
+- Si se agrega un nuevo modelo ejecutar annotate para anotar los campos que conforman esa tabla en la parte superior del modelo
```
Comment (EduardoGHdez) [inline README.md:70]: We could avoid having to remember this extra step by automating this process 😄 
https://github.com/ctran/annotate_models#configuration-in-rails

File: lib/tasks/auto_annotate_models.rake [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# NOTE: only doing this in development as some production environments (Heroku)
```
Comment (EduardoGHdez) [inline lib/tasks/auto_annotate_models.rake:1]: Ohh you just added it 👌🏽 

I think you don't need to remember this any longer, and also keep the README as it was

File: README.md (hunk 1 above)
Comment (YilKanda) [reply README.md:70]: I agree with what @EduardoGHdez  mentions, likewise, I would just like to suggest that if you are indicating that a library be executed, you can put the command to execute.

File: app/views/groups/show.html.erb [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
   <%= @group.name %>
 </p>
-
 <p>
   <strong>Group Description:</strong>
   <%= @group.description %>
 </p>
+<p>
+  <strong>Group Amount:</strong>
+  <%=number_to_currency( @group.amount,locale: :en) %>
```
Comment (fabiannabi) [inline app/views/groups/show.html.erb:16]: Its a small one, but just check the spaces in the `( )` at the beginning

File: README.md (hunk 1 above)
Comment (jestradadeveloper) [reply README.md:70]: The command is annotate, its mentioned in the comment

Comment (jestradadeveloper) [reply README.md]: @EduardoGHdez  as you mention after this comment i already add the tasks for these actions, I've made an amend removing the  unnecessary comment at the readme file. Thanks for your CR.

File: app/views/groups/show.html.erb (hunk 3 above)
Comment (jestradadeveloper) [reply app/views/groups/show.html.erb]: I done this. Now there isn't any space at the beginning of ( ) . Thanks for your CR

File: lib/tasks/auto_annotate_models.rake (hunk 2 above)
Comment (jestradadeveloper) [reply lib/tasks/auto_annotate_models.rake:1]: You are right. Thanks

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by adrianMRJ — APPROVED** [review]
LGTM!

**Review by CUBOFIG — APPROVED** [review]
LGTM! :dango: :dagger:

**Pull request merged**